homeassistant/components/keymitt_ble/entity.py

```diff
@@ -0,0 +1,34 @@
+"""MicroBot class."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.bluetooth.passive_update_coordinator import (
+    PassiveBluetoothCoordinatorEntity,
+)
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo
+
+from .const import MANUFACTURER
+
+
+class MicroBotEntity(PassiveBluetoothCoordinatorEntity):
+    """Generic entity for all MicroBots."""
+
```

Comment:
```suggestion
    coordinator: MicroBotDataUpdateCoordinator
```

black will complain about this suggesiton